### PR TITLE
Sean/'In the words of our employees' / testimonial section

### PR DESCRIPTION
**Pull request by sean-binary**
# Description

Summary
- Added testimonial section along with mobile layout

Changes:
- Added back the testimonial section
- Updated with new people
- Added layout for mobile

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sara-fs — DISMISSED** [review]

File: src/pages/careers/_employee-testimonial-carousel.js [hunk 1]
```diff
@@ -193,31 +272,44 @@ EmployeeSlide.propTypes = {
...
     name: 'Negar Naghshbandi, Front-end Developer & Team Lead',
     img_alt: localize('Negar - Front-end Developer & Team Lead'),
     img_path: NegarImage,
     quote:
         'The most exciting thing for me is the culture of the company and the people I work with. I learn something new everyday and I can pair-program with anyone when needed because everyone is approachable and eager to help.',
+    index: 1,
+}
+const ahmad = {
+    name: 'Ahmad Aizuddin Disaster, Recovery Analyst',
+    img_alt: localize('Ahmad - Recovery Analyst'),
```
Comment (mustofa-binary) [inline src/pages/careers/_employee-testimonial-carousel.js:285]: please remove localize function here

File: src/pages/careers/_employee-testimonial-carousel.js [hunk 2]
```diff
@@ -193,31 +272,44 @@ EmployeeSlide.propTypes = {
...
     index: 2,
 }
-const mahdi = {
-    name: 'Mahdi Pourziaei, Front-end Developer',
-    img_alt: localize('Mahdi - Front-end Developer'),
-    img_path: MahdiImage,
-    quote: `Two words: 'autonomy' and 'friendly'. Working at Deriv has been full of growth as I get to pick my own challenges and see them through, and it really wouldn’t feel as empowering without the friendly culture. I’m happy to be amongst all the brilliant people here.`,
+const aditya = {
+    name: 'Lalitaditya Addanki, DevOps Engineer',
+    img_alt: localize('Lalitaditya - DevOps Engineer'),
```
Comment (mustofa-binary) [inline src/pages/careers/_employee-testimonial-carousel.js:292]: here too, and others below

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/pages/careers/_employee-testimonial-carousel.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/careers/_employee-testimonial-carousel.js:285]: Alright noted

File: src/pages/careers/_employee-testimonial-carousel.js (hunk 2 above)
Comment (sean-binary) [reply src/pages/careers/_employee-testimonial-carousel.js:292]: Alright noted

**Review by mustofa-binary — APPROVED** [review]

**Pull request closed without merging**